recipes/theora/all/conanfile.py

```diff
@@ -1,182 +1,96 @@
-from conan.tools.microsoft import msvc_runtime_flag
-from conans import ConanFile, MSBuild, AutoToolsBuildEnvironment, tools
-import functools
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.microsoft import is_msvc
+from conan.errors import ConanException
 import os
-import re
-import shutil
-import stat
-
-required_conan_version = ">=1.36.0"
 
+required_conan_version = ">=1.52.0"
 
 class TheoraConan(ConanFile):
     name = "theora"
-    description = "Theora is a free and open video compression format from the Xiph.org Foundation"
+    license = "BSD-3-Clause"
```

Comment:
https://github.com/xiph/theora/blob/master/LICENSE

There should be a license ref here https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-license-should-i-use-for-a-custom-project-specific-license